packages/wrangler/src/pages/functions.tsx

```diff
@@ -0,0 +1,86 @@
+import { existsSync, readFileSync, writeFileSync } from "node:fs";
+import { FatalError } from "../errors";
+import {
+	isRoutesJSONSpec,
+	optimizeRoutesJSONSpec,
+} from "./functions/routes-transformation";
+import { pagesBetaWarning } from "./utils";
+import type { ArgumentsCamelCase, Argv } from "yargs";
+
+type OptimizeRoutesArgs = {
+	"routes-path": string;
+	"output-routes-path": string;
+};
+
+export function OptimizeRoutesOptions(yargs: Argv): Argv<OptimizeRoutesArgs> {
+	return yargs
+		.options({
+			"routes-path": {
+				type: "string",
+				demandOption: true,
+				description: "The location of the _routes.json file",
+			},
+		})
+		.options({
+			"output-routes-path": {
+				type: "string",
+				demandOption: true,
+				description: "The location of the optimized output routes file",
+			},
+		})
+		.epilogue(pagesBetaWarning);
```

Comment:
Rather than doing this here, let's log the pagesBetaWarning inside the handler similar to routes:
https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/pages/build.tsx?at=a3e654f#L95-L98
```typescript
if (!isInPagesCI) {
		// Beta message for `wrangler pages <commands>` usage
		logger.log(pagesBetaWarning);
	}
```
We don't want to print this warning when using in CI - it's only "beta" when used outside of CI.
(Forgot to put this in the requirements, sorry!)